README.md

```diff
@@ -41,6 +41,26 @@ console.log(config);
 - It is possible to change `%` to any other character. Just use `replaceEnv` configuration option.
 - It is possible to use default values when environmental variable is not set.
 
+### Docker secret replacement
+
+/tmp/config.json:
+``` javascript
+{ docks1: "%{DOCKER_SECRET1}", docks2: "%{DOCKER_SECRET2|def}" }
```

Comment:
there should be probably '#' instead of '%' in this example ;)

Reply:
Thanks!, will correct it.